superset/viz.py

```diff
@@ -1155,6 +1153,9 @@ def process_data(self, df, aggregate=False):
         if fd.get("granularity") == "all":
             raise Exception(_("Pick a time granularity for your time series"))
 
+        if df.empty:
```

Comment:
Short circuiting as `pivot_table` will throw an exception if the `pandas.DataFrame` is empty.